src/foam/u2/crunch/EasyCrunchWizard.js

```diff
@@ -70,6 +70,14 @@ foam.CLASS({
       documentation: `
         Require all sections to be valid to invoke wizard completion (done button).
       `
+    },
+    {
+      class: 'Boolean',
+      name: 'disableGrantedEdit',
```

Comment:
The naming of this variable is not intuitive - even with an understanding of what it's controlling.